EKF/control.cpp

```diff
@@ -1161,10 +1161,11 @@ void Ekf::controlHeightFusion()
 			Vector2f rng_hgt_innov_gate;
 			Vector3f rng_hgt_obs_var;
 			// use range finder with tilt correction
-			_rng_hgt_innov(2) = _state.pos(2) - (-math::max(_range_sensor.getRange() * _range_sensor.getCosTilt(),
+			_rng_hgt_innov(2) = _state.pos(2) - (-math::max(_range_sensor.getDistBottom(),
 							 _params.rng_gnd_clearance)) - _hgt_sensor_offset;
 			// observation variance - user parameter defined
-			rng_hgt_obs_var(2) = fmaxf((sq(_params.range_noise) + sq(_params.range_noise_scaler * _range_sensor.getRange())) * sq(_range_sensor.getCosTilt()), 0.01f);
+			rng_hgt_obs_var(2) = fmaxf(sq(_params.range_noise)
+						   + sq(_params.range_noise_scaler * _range_sensor.getDistBottom()), 0.01f);
```

Comment:
Do we really want to scale the range noise with the distance below the drone and not the measured distance?

Reply:
I am not sure what is better.

Reply:
I guess there's 2 things adding noise:

1) Sensor characteristics
2) Terrain unevenness

1) scales with measured distance, 2) scales with distance to ground and vehicle orientation change rate (looking at different points on the ground).

Not sure what the right solution is here, or which component dominates.